src/cli.rs

```diff
@@ -391,6 +392,10 @@ pub enum Command {
     /// Commands related to updating rover
     Update(command::Update),
 
+    /// Commands for persisted queries
+    #[command(alias = "pq")]
```

Comment:
Perhaps `visible_alias` instead, for discoverability?

Do we want this to have `hide = true` for now?